### DISCUSSION TITLE
Using Azure DevOps Caching for react native windows based PR builds

### DISCUSSION THREAD

**Discussion by JoshyAbraham** [microsoft/react-native-windows#14203] [Q&A] [answered]
Our team is exploring the possibility of using build pipeline caching to speed up PR builds for a windows react native based project.  Our intention is to cache the react native binaries so that we don't need to build the react native libraries on each PR build. our pipelines do a clean fresh fetch to build each PR and is not using incremental fetch. We've reviewed the documentation here: [Azure DevOps Pipeline Caching](https://learn.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops).
Has anyone tried implementing this for windows react based projects, and if so, have you had success leveraging caching to improve build performance?

**Comment by chrisglein** [comment, marked as answer]
We haven't looked into that, however we do know that build times are long and our plan for this is by leaning into pre-build nuget binaries as the default starting with RNW on Meta's new architecture (and you can try this out today!). That should give significant wins in your build time, and will become the default soon.
